docs: add support docs back (#4033)
neondatabase/website: content/docs/introduction/support.md

To ensure efficient resolution of issues, customers are expected to (1) provide detailed information about the issue, (2) cooperate with the Support team during troubleshooting, and (3) utilize available self-service resources for basic inquiries.

<a id="enterprise-support"></a>

## Legacy enterprise support

> Customers on our legacy enterprise plans can view support plan details in the following dropdown.

Author: ```suggestion
Customers on a legacy [Enterprise plan](/docs/introduction/legacy-plans#enterprise-plan-legacy) can view support plan details in the following dropdown.
```
